commit baae3d8115f61eb269451824fced2a68e5c9f23a
Author:     Yuhong Wen <yuhongw@nvidia.com>
AuthorDate: Wed Apr 3 17:33:56 2024 -0400
Commit:     GitHub <noreply@github.com>
CommitDate: Wed Apr 3 17:33:56 2024 -0400

    Fixed the authz and site_security check for check_resource command. (#2462)
    
    Co-authored-by: Yuan-Ting Hsieh (謝沅廷) <yuantingh@nvidia.com>
---
 nvflare/private/fed/server/server_engine.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index 955861ec..cc0c3a04 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -760,7 +760,7 @@ class ServerEngine(ServerEngineInternalSpec):
     def _make_message_for_check_resource(self, job, resource_requirements, fl_ctx):
         request = Message(topic=TrainingTopic.CHECK_RESOURCE, body=resource_requirements)
         request.set_header(RequestHeader.JOB_ID, job.job_id)
-        request.set_header(RequestHeader.REQUIRE_AUTHZ, "true")
+        request.set_header(RequestHeader.REQUIRE_AUTHZ, "false")
         request.set_header(RequestHeader.ADMIN_COMMAND, AdminCommandNames.CHECK_RESOURCES)
 
         set_message_security_data(request, job, fl_ctx)
